.github/workflows/deploy-webhook.yml

```diff
@@ -0,0 +1,16 @@
+name: Autodeployment Webhook
+
+on:
+  workflow_call:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Invoke autodeployment webhook for staging
+        uses: distributhor/workflow-webhook@v2
+        env:
+          webhook_type: 'json-extended'
+          webhook_url: https://dev.telescope.cdot.systems/deploy
+          webhook_secret: ${{ secrets.WEBHOOK_SECRET }}
```

Comment:
@manekenpix I'm going to need to put our `WEBHOOK_SECRET` on staging and production into the GitHub Actions Secrets.  Can you help me when you have a min to figure out what it is?

Reply:
Secrets now added, thank you @manekenpix!